
> guincho-rs@0.0.0 dev /home/ubuntu/rsapp
> vite


  VITE v6.3.5  ready in 1113 ms

  ➜  Local:   http://localhost:5173/
  ➜  Network: use --host to expose
  ➜  press h + enter to show help

  #  Node.js[4665]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0x7f602da45495  [/lib/x86_64-linux-gnu/libc.so.6]
 4: 0x7f602da45610 on_exit [/lib/x86_64-linux-gnu/libc.so.6]
 5: 0xe98a4f node::DefaultProcessExitHandlerInternal(node::Environment*, node::ExitCode) [node]
 6: 0xf01e4f node::Environment::Exit(node::ExitCode) [node]
 7: 0x1d755e2  [node]

  #  Node.js[4665]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0xf42982 node::TrapWebAssemblyOrContinue(int, siginfo_t*, void*) [node]
 4: 0x7f602da42520  [/lib/x86_64-linux-gnu/libc.so.6]
Aborted
 ELIFECYCLE  Command failed with exit code 134.

  #  Node.js[4653]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0x7f132c045495  [/lib/x86_64-linux-gnu/libc.so.6]
 4: 0x7f132c045610 on_exit [/lib/x86_64-linux-gnu/libc.so.6]
 5: 0xe98a4f node::DefaultProcessExitHandlerInternal(node::Environment*, node::ExitCode) [node]
 6: 0xf01e4f node::Environment::Exit(node::ExitCode) [node]
 7: 0x1d755e2  [node]

  #  Node.js[4653]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0xf42982 node::TrapWebAssemblyOrContinue(int, siginfo_t*, void*) [node]
 4: 0x7f132c042520  [/lib/x86_64-linux-gnu/libc.so.6]

  #  Node.js[4642]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0x7f839ea45495  [/lib/x86_64-linux-gnu/libc.so.6]
 4: 0x7f839ea45610 on_exit [/lib/x86_64-linux-gnu/libc.so.6]
 5: 0xe98a4f node::DefaultProcessExitHandlerInternal(node::Environment*, node::ExitCode) [node]
 6: 0xf01e4f node::Environment::Exit(node::ExitCode) [node]
 7: 0x1d755e2  [node]

  #  Node.js[4642]: void node::ResetStdio() at ../src/node.cc:760
  #  Assertion failed: !(err != 0) || (err == -1 && (*__errno_location ()) == 1)

----- Native stack trace -----

 1: 0xf95cf7 node::Assert(node::AssertionInfo const&) [node]
 2: 0xf42264  [node]
 3: 0xf42982 node::TrapWebAssemblyOrContinue(int, siginfo_t*, void*) [node]
 4: 0x7f839ea42520  [/lib/x86_64-linux-gnu/libc.so.6]
